src/util/get_from_db.py

```
@@ -34,7 +35,37 @@ def get_user_role_level(role_name: str) -> int:
+        raise HTTPException(503, detail=f"config entry '{key}' not configured")                                           
+    return kv_entry                                                                                                   
+                                                                                                                          
+                                                                                                                        
+                                                                                                                        
+def update_kv_entry(                                                                                                  
+    session: SessionDep,                                                                                                  
+    *,                                                                                                                    
+    key: str,                                                                                                             
+    value: str,                                                                                                           
+    actor_role: int,                                                                                                      
+    min_role: int | None = None,                                                                                          
```

Ruff (0.13.3)

Unused function argument: `min_role`

(ARG001)